tests/integration_tests/charts/api_tests.py

```diff
@@ -83,120 +83,114 @@ class TestChartApi(ApiOwnersTestCaseMixin, InsertChartMixin, SupersetTestCase):
 
     @pytest.fixture(autouse=True)
     def clear_data_cache(self):
-        with app.app_context():
```

Comment:
The app context is not needed throughout this file. My _hunch_ is this is being setup indirectly from Flask-Testing via the `TestCase` class.